packages/core/core/src/Strapi.ts

```diff
@@ -481,16 +481,16 @@ class Strapi extends Container implements StrapiI {
   }
 
   async bootstrap() {
-    const contentTypes = [
-      coreStoreModel,
-      webhookModel,
-      ...Object.values(this.contentTypes),
-      ...Object.values(this.components),
-    ];
-
     this.db = await Database.init({
       ...this.config.get('database'),
-      models: utils.transformContentTypesToModels(contentTypes),
+      models: [
+        utils.transformContentTypesToModels([
+          ...Object.values(this.contentTypes),
+          ...Object.values(this.components),
+        ]),
+        coreStoreModel,
+        webhookModel,
+      ],
```

Comment:
the ideea is to provide an interface to register models dynamically here too right?

Reply:
yes eventually :)